### ISSUE TITLE
Document how Fiji versioning works

### ISSUE THREAD

**Issue by qgreen63** [imagej/imagej.github.io#287]
Downloading the latest Mac release for Fiji actually downloads the 2.9.0 version, despite the latest release being 2.11.0

**Comment by ctrueden** [comment]
@qgreen63 Yes, this is intended. The developer-facing 2.10.0 and 2.11.0 releases have not made their way to the core update site yet, and the latest download matches the contents of the core update site. If you want to follow the progress, see [this Image.sc forum topic](https://forum.image.sc/t/plugin-maintainers-can-you-test-fiji-2-11-0/78852). Unfortunately, there are a couple of issues that need to be ironed out before we can upload 2.11.0 to users.

**Comment by ctrueden** [comment]
@qgreen63 Good question. I don't think so, unfortunately. Where would you suggest I write up an explanation? The page https://imagej.net/software/fiji/downloads could work...

How did you learn that 2.11.0 is the latest release? Due to use of the `sc.fiji:fiji` Maven artifact, perhaps?

**Comment by qgreen63** [comment]
I saw the version number in the "Vital Statistics" pane on this page: https://imagej.net/software/fiji/

That page would work. I am attempting to automatically deploy and update fiji on a fleet of macs. The builtin updater requires permissions that the users do not have, so the new versions need to be downloaded manually. The app itself does not have version info where mac apps usually do (which should probably be opened as an issue itself). I can scrape a web page for version info to figure it out manually, but without a release log page or any documented version info I can find that is impossible to do automatically.

**Comment by qgreen63** [comment]
Yes, looking at the last modified date is sufficient to decide if an update is needed, however the solution I am using for the actual installs uses version numbers to determine if the downloaded package is indeed a newer version that should be installed. While I _could_ inject the last modified date in to the versioning info and use that as the version number, I would prefer to use the actual version number...
